docs/_articles/en/getting-started/recommended-extensions.md

```diff
@@ -3,14 +3,20 @@ title: Recommended Extensions
 lang: en
 ---
 
-In addition to the extension pack shipped by Salesforce, the following extensions are useful for Salesforce developers.  We also package the Extension pack with some of the most-used community extensions in the [expanded extension pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-expanded). 
+In addition to the extension pack shipped by Salesforce, the following extensions are useful for Salesforce developers. We also package the Extension pack with some of the most-used community extensions in the [expanded extension pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-expanded).
 
 ## Apex PMD
 
 Allows you to run Apex static analysis directly in VS Code on Apex and Visualforce files.
 
 <https://marketplace.visualstudio.com/items?itemName=chuckjonas.apex-pmd>
 
+## Apex Log Analyzer
+
+An analyzer for Salesforce debug logs aimed at making performance analysis much easier and quicker. You may also find it generally useful for quickly understanding how your code is executing.
```

Comment:
It's also useful for quickly understanding how your code is excuting.

(But looks like you already merged - do you still need me to approve?)